### PR TITLE
chore: 🧹 enabled experiment code for scanning reminder for all [ROAD-503]

### PR DESCRIPTION
**Pull request by bastiandoetsch**
don't ask Amplitude for this experiment anymore, only use setting if it was shown. If not, show it, update the setting and don't show it again.

### CODE REVIEW

File: src/main/kotlin/io/snyk/plugin/services/SnykApplicationSettingsStateService.kt [hunk 1]
```diff
@@ -56,7 +56,6 @@ class SnykApplicationSettingsStateService : PersistentStateComponent<SnykApplica
     var lastTimeFeedbackRequestShown: Date = Date.from(Instant.now())
     var showFeedbackRequest = true
 
-    // experiment section
     var scanningReminderWasShown: Boolean = false
```
Comment (michelkaporin) [inline src/main/kotlin/io/snyk/plugin/services/SnykApplicationSettingsStateService.kt:59]: Do we still need this bool?

**Comment by bastiandoetsch** [comment]
@ArtsiomCh could you review this quickly please, as we could take it then into the release :)

**Review by ArtsiomCh — APPROVED** [review]
LGTM

**Comment by ArtsiomCh** [comment]
@bastiandoetsch why close? at least `userToken.isNotBlank()` check is needed before allow to run the scan from reminder

**Comment by bastiandoetsch** [comment]
Reopened

File: src/main/kotlin/io/snyk/plugin/SnykPostStartupActivity.kt [hunk 2]
```diff
@@ -78,11 +78,6 @@ class SnykPostStartupActivity : StartupActivity.DumbAware {
         LOG.info("Loading variants for all amplitude experiments")
         val experimentUser = ExperimentUser(publicUserId)
         service<AmplitudeExperimentService>().fetch(experimentUser)
-
-        if (service<AmplitudeExperimentService>().isShowScanningReminderEnabled()) {
-            SnykBalloonNotifications.showScanningReminder(project)
```
Comment (ArtsiomCh) [inline src/main/kotlin/io/snyk/plugin/SnykPostStartupActivity.kt:83]: Shell we remove `SnykBalloonNotifications.showScanningReminder` and correspondent code as well? Or we planning to reenable it later?
cc @pavel-snyk

Comment (bastiandoetsch) [reply src/main/kotlin/io/snyk/plugin/SnykPostStartupActivity.kt:83]: I'll just remove it and merge - we can always get it back from git.

**Pull request merged**